rpc/api.proto

```diff
@@ -2,6 +2,8 @@ syntax = "proto3";
 
 package walletrpc;
 
+option go_package = "github.com/decred/dcrwallet/rpc/walletrpc";
```

Comment:
the package path is `decred.org/dcrwallet/rpc/walletrpc`.

Reply:
Thanks, fixed.